src/main/kotlin/com/vk/admstorm/AdmStormStartupActivity.kt

```diff
@@ -67,26 +68,36 @@ class AdmStormStartupActivity : ProjectActivity {
         Logger.setFactory(AdmStormLoggerFactory(sentry, defaultLoggerFactory))
     }
 
+    private fun hasNewVersion(pluginId : PluginId): Boolean {
```

Comment:
Let's put the check for new versions in a separate class?

Reply:
Done